src/views/InteractiveMap/CardDialogDesign.vue

```diff
@@ -163,6 +163,17 @@
       @change="addReaction"
     ></v-autocomplete>
 
+    <v-icon :disabled="!model">add</v-icon>
+    <v-btn @click.stop="isReactionDialogVisible = true" :disabled="!model"
+      >Add arbitrary reaction
```

Comment:
Maybe the label could rather be something like "Define your own reaction" to make more sense to the user